packages/wallets/src/components/WalletTourGuide/WalletTourGuide.tsx

```diff
@@ -1,31 +1,71 @@
-import React from 'react';
-import Joyride, { CallBackProps } from 'react-joyride';
+import React, { useEffect } from 'react';
+import Joyride, { ACTIONS, CallBackProps } from 'react-joyride';
+import { useLocalStorage, useReadLocalStorage } from 'usehooks-ts';
+import { useActiveWalletAccount, useAuthorize, useAvailableWallets, useWalletAccountsList } from '@deriv/api';
+import useDevice from '../../hooks/useDevice';
+import { WalletText } from '../Base';
 import { TooltipComponent, tourStepConfig } from './WalletTourGuideSettings';
 import './WalletTourGuide.scss';
 
-type TProps = {
-    isStarted?: boolean;
-    setIsStarted?: (value: boolean) => void;
-};
+const WalletTourGuide = () => {
+    const key = 'walletsOnboarding';
+    const startValue = 'started';
+    const [walletsOnboarding, setWalletsOnboarding] = useLocalStorage(key, useReadLocalStorage(key));
+    const { isMobile } = useDevice();
+
+    const { isFetching, isLoading, isSuccess, switchAccount } = useAuthorize();
+    const { data: wallets } = useWalletAccountsList();
+    const { data: activeWallet } = useActiveWalletAccount();
+    const { data: availableWallets } = useAvailableWallets();
+
+    const fiatWalletLoginId = wallets?.[0]?.loginid;
```

Comment:
is first wallet always fiat? what if order changes? should it be found by flags or something more reliable?

Reply:
Yes, it's sorted in useWalletAccountsList hook:
`// Sort wallet accounts alphabetically by fiat, crypto, then virtual`

Reply:
We shouldn't rely on hidden/non-explicit behaviours. If order within hook changes, the above will simply no longer be correct and might result in elusive bug, especially hard to catch given its only visible during onboarding (so I'm guessing only fresh browser / fresh account).

Reply:
if fiat wallet specifically needs to be selected, then fiat wallet should be explicitly selected from list (unless first wallet is needed no matter the type)

Reply:
I'm sure it's the main business requirement that fiat wallet must be first so I'm sure it will not be changed. If yes, it means we have to change many things in onboarding flow, not just switching to first wallet also we have to pass number of fiat wallet to ondoarding steps to highlight proper wallet.